src/SingleBookingPage/bookingItem/AccordionHeader.js

```diff
@@ -0,0 +1,24 @@
+// @flow
```

Comment:
What about renaming this component and file to `AccordionTitle` or sth similar? IMHO the "header" and "body" of the accordion should be the visible part when untoggled and toggled, respectively.

Reply:
Ok, makes sense